src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -385,6 +385,12 @@ private function getToolbarButtons()
                 'icon' => 'add_circle_outline',
                 'class' => 'btn-outline-primary',
             ];
+
+            $toolbarButtons['list_v2'] = [
+                'href' => $this->generateUrl('admin_products_v2_index'),
+                'desc' => $this->trans('List on experimental page', 'Admin.Catalog.Feature'),
```

Comment:
This wording is new, where does it appear? 🤔

Reply:
Hi @Julievrz Indeed this wording is new, I used the same principle as the buttons we used in the product page edition.
This one is visible on the legacy list if you want to switch to the new one. However, it should be removed before the final release since the full migration of the product page should be finished by then.

Reply:
Ok, I see, thanks! Lgtm :-)